dags/app/cloudflare_bot/dag.py

```diff
@@ -0,0 +1,74 @@
+"""
+Create bot network traffic
+"""
+import logging
+from datetime import datetime, timedelta
+
+import requests
+from airflow.sdk import Variable, dag, task
+
+DEFAULT_ARGS = {
+    "owner": "Henry Lee",
+    "depends_on_past": False,
+    "start_date": datetime(2025, 1, 1),
+    "retries": 2,
+    "retry_delay": timedelta(minutes=5),
+}
+
+logger = logging.getLogger(__name__)
+
+
+@dag(
+    default_args=DEFAULT_ARGS,
+    schedule="@hourly",
+    max_active_runs=1,
+    catchup=False,
+)
+def PYCONTW_ETL_BOT_v1():
+
+    @task
+    def GET_TOP_WEBSITES() -> list[str]:
+        """Call Cloudflare Radar and return a list of the top-100 domains.
+
+        Docs: https://developers.cloudflare.com/api/resources/radar/subresources/ranking/methods/top/
+        """
+        token = Variable.get("CLOUDFLARE_RADAR_API_TOKEN")
+
+        url = "https://api.cloudflare.com/client/v4/radar/ranking/top"
+        params = {"limit": 100}  # 100 is the maximum allowed
+        headers = {"Authorization": f"Bearer {token}"}
+
+        response = requests.get(url, headers=headers, params=params, timeout=10)
+        response.raise_for_status()
+        data = response.json()
+
+        # Radar API response: {"result":{"top_0":[{"domain":"google.com", ...}, ...]}}
+        domains = [item["domain"] for item in data.get("result", {}).get("top_0", [])]
+
+        logger.info("Fetched %d domains from Cloudflare Radar", len(domains))
+        return domains
+
+    @task
+    def REQUEST_EACH_WEBSITE(domains: list[str]):
```

Comment:
```suggestion
    def ping_each_website(domains: list[str]) -> None:
```

looks like we're pinging these sides?